gotime/go-time-5.md

```diff
@@ -106,7 +106,7 @@ Alright everybody, welcome back for another episode of Go Time. This is episode
 
 **Erik St. Martin:** Yeah, and I think that the vendor stuff is probably early for that stuff to kind of get pulled in, at least adopted the Go team, right? Because it's only been recently that there's been this agreement that maybe the Go tooling should handle vendoring more, so it will be interesting to see how much they think should be pulled in. But I imagine we'll start seeing tools like Vim-Go and all that jazz incorporating in this, in something like the meta tooling.
 
-**Carlisia Pinto:** Exactly.
+**Sarah Adams:** Exactly.
```

Comment:
This still sounds like Carlisia to me, can you check again?

Reply:
Hey @pnevares , It is Carlisia, Thanks for the review :)